shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/API/CLI.hs

```diff
@@ -0,0 +1,97 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module Shelley.Spec.Ledger.API.CLI
```

Comment:
Are these just for the CLI, or can we merge into `Wallet`, seeing as these are needed for some Wallet functions?

Reply:
I don't know. If you think these would be better placed in Wallet, let me know!